gui-libs/wpebackend-fdo/wpebackend-fdo-1.14.2-r1.ebuild

```diff
@@ -27,7 +27,7 @@ BDEPEND="
 "
 
 PATCHES=(
-	"${FILESDIR}"/${P}-c23-fix.patch
+	"${FILESDIR}"/wpebackend-fdo-1.14.3-c23-fix.patch
```

Comment:
as you prefer to apply it for both (fine), we often use the older version it will apply for the patch -> wpebackend-fdo-1.14.2-c23-fix.patch

Then, you can use "${FILESDIR}"/${P}-c23-fix.patch for wpebackend-fdo-1.14.2-r1.ebuild